app/assets/javascripts/darkswarm/services/enterprise_modal.js.coffee

```diff
@@ -3,6 +3,7 @@ Darkswarm.factory "EnterpriseModal", ($modal, $rootScope)->
   new class EnterpriseModal
     open: (enterprise)->
       scope = $rootScope.$new(true) # Spawn an isolate to contain the enterprise
+      scope.embedded_layout = window.location.search.indexOf("embedded_shopfront=true") != -1
```

Comment:
Added this because I noticed that links in the enterprise model launched from the map weren't opening in a new tab. It turns out this modal inherits from a different angular controller to the rest of the page.